mpi4jax/collective_ops/allreduce.py

```diff
@@ -52,7 +52,7 @@ def Allreduce(x, op, comm=_MPI.COMM_WORLD, token=None):
             This result can be ignored if result forces a data dependency.
     """
     if token is None:
-        token = create_token(stop_gradient(x))
+        token = create_token(x)
 
```

Comment:
`create_token` already calls `stop_gradient` internally, see https://github.com/google/jax/blob/8f0425d4120a8a3858914c1f4c73c50d4b484ae6/jax/_src/lax/lax.py#L5858
so I think we can omit it here?

Reply:
I think that's new. 
Yes, we can drop it.